### DISCUSSION TITLE
Can we return username and password from credentials_script while auth?

### DISCUSSION THREAD

**Discussion by kbhalaki** [NetApp/harvest#2866] [General]
We want harvest to get both username and password from credentials script at runtime.  The [doc ](https://netapp.github.io/harvest/24.02/configure-harvest-basic/#example) mentions that addr and username are inputs to credentials_script but the example does not container username. Can someone please clarify?

**Comment by rahulguptajss** [comment]
Discussed over [discord](https://discord.com/channels/855068651522490400/1235499249433837630)
